dayrepo/snippets/views.py

```diff
@@ -154,3 +152,33 @@ def post(self, request):
 
 
 snippet_post = SnippetView.as_view()
+
+# checklist入力画面表示/POST処理
+class ChecklistView(View):
+    # 新規入力画面
+    def get(self, request):
+        # 投稿ボタンで投稿ページへ
```

Comment:
```suggestion
        # 点検・投稿ボタンで点検ページへ
```

できればこっち(↓)のがシンプル 笑
フロントエンドのコンポーネント(ボタン)とバックエンドのAPIの処理はあんまり関係ないのと、わんちゃんヘッダーのボタン以外でもこの get を使いたいケースがあるかもしれない（ってか、確か日報入力画面であったような?）ので、html について言及するべきではない派
```suggestion
        # 点検ページへ
```

Reply:
画面推移は次のPRで変わるからここも次にと思ってたけど、まぁいいかな？
次のPR次第では再修正もありますが、採用しておきます。